README.md

```diff
@@ -45,7 +45,7 @@ The custom resources needed to define a pipeline are listed below:
 In short, in order to create a pipeline, one does the following:
 * Create custom or install [existing](https://github.com/tektoncd/catalog) reusable `Tasks`
 * Create a `Pipeline` and `PipelineResources` to define your application's delivery pipeline
-* Create a `PersistentVolumeClaim` to provide the volume/filesystem for pipeline execution
+* Create a `PersistentVolumeClaim` to provide the volume/filesystem for pipeline execution or provide a `VolumeClaimTemplate` which creates a `PersistentVolumeClaim`
```

Comment:
I think we need to change this sentence bit because we are not creating `PersistentVolumeClaim` now

Reply:
It's just extra information. Anyway even `VolumeClaimTemplate` creates a PVC. I think it's ok to keep it :)